cachi2/core/package_managers/yarn/utils.py

```diff
@@ -1,6 +1,8 @@
 import os
 import subprocess
-from typing import Optional
+from typing import Optional, Union
```

Comment:
**Commit message**:

s/unifies/unified
s/handling of/handling of Yarn
s/and/with a new

Reply:
Done.